tests/indent/python/branches.py

```diff
@@ -23,7 +23,7 @@ def foo(x, y):
     b)
 
 if (a and
-    b):
+        b):
```

Comment:
actually, my python linter recommends this way were you indent the second line of the parenthesis.